lib/client/tncon/tncon.c

```diff
@@ -0,0 +1,164 @@
+// +build windows,cgo
+
+/*
+* Author: Ray Hayes <ray.hayes@microsoft.com>
```

Comment:
We've modified these files, right? If so, it would be nice to indicate that somehow.

Reply:
I've added a line:

    Modifications for use in Teleport Copyright 2021 Gravitational, Inc.

It's a bit of an odd situation so I'm not sure if we have some preferred syntax for things like this.